core-bundle/src/DataCollector/ContaoDataCollector.php

```diff
@@ -145,7 +145,7 @@ private function addSummaryData(): void
             'framework' => $framework,
             'models' => $modelCount,
             'frontend' => isset($GLOBALS['objPage']),
-            'preview' => \defined('BE_USER_LOGGED_IN') && true === BE_USER_LOGGED_IN,
+            'preview' => \defined('BE_USER_LOGGED_IN') && true === \constant('BE_USER_LOGGED_IN'),
```

Comment:
Why do we need `\constant()` here? We have already checked `\defined()`.

Reply:
Because this way PHPStan won't complain about 'Strict comparison using === between true and false will always evaluate to false'. We ignored this previously but imo it doesn't hurt to use the constant function and rather have less exceptions.

Reply:
I think I like ignoring this error better than using `constant()`

Reply:
Reverted in 1eb47e970be6eaa5a225104a7a7ba6f073b2bf63

Reply:
Found a better way: see https://phpstan.org/config-reference#constants

Changed in 63da7dd9541674f4f699d8a686cceab0b8c721b6